[2025-09-16T16:09:37.308+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_pipeline_postgres.transform manual__2025-09-16T16:09:32.190871+00:00 [queued]>
[2025-09-16T16:09:37.313+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_pipeline_postgres.transform manual__2025-09-16T16:09:32.190871+00:00 [queued]>
[2025-09-16T16:09:37.313+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-09-16T16:09:37.321+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform> on 2025-09-16 16:09:32.190871+00:00
[2025-09-16T16:09:37.324+0000] {standard_task_runner.py:57} INFO - Started process 256 to run task
[2025-09-16T16:09:37.326+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crypto_pipeline_postgres', 'transform', 'manual__2025-09-16T16:09:32.190871+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/coleta_dados.py', '--cfg-path', '/tmp/tmpmtu_6jrr']
[2025-09-16T16:09:37.328+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask transform
[2025-09-16T16:09:37.356+0000] {task_command.py:415} INFO - Running <TaskInstance: crypto_pipeline_postgres.transform manual__2025-09-16T16:09:32.190871+00:00 [running]> on host 529eb878ec8c
[2025-09-16T16:09:37.398+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_pipeline_postgres' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T16:09:32.190871+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-16T16:09:32.190871+00:00'
[2025-09-16T16:09:37.410+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-09-16T16:09:37.416+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=crypto_pipeline_postgres, task_id=transform, execution_date=20250916T160932, start_date=20250916T160937, end_date=20250916T160937
[2025-09-16T16:09:37.458+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-16T16:09:37.472+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
